src/controllers/userController.js

```diff
@@ -106,5 +106,32 @@ const getProfile = async (request, response) => {
   });
 };
 
-const userController = { getProfile, signUp, login };
-export default userController;
+/**
+ * @description updates the profile of a user
+ * @param {object} request express request object
+ * @param {object} response express response object
+ * @param {object} next express next argument
+ * @returns {json} json
+ */
+const editProfile = async (request, response) => {
+  const { body, user } = request;
```

Comment:
what if there are other parameters sent in the body that doesn't exist in your user table

Reply:
An error would occur with the response 'the field(s) entered are invalid'

Reply:
Can you point me to the line of code that handles that?

Reply:
On line 123 and line 132 of the userController.js